app/templates/src/index.adoc

```diff
@@ -0,0 +1,58 @@
+= <%= title %>
+:!sectids:
+:imagesdir: images
+:source-highlighter: coderay
+:coderay-css: style
+
+== Images
+image::bespoke-logo.jpg[]
+
+== Keyboard Interaction
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-keys[bespoke-keys]
+
+== Touch Interaction
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-scale[bespoke-scale]
+
+== Bullet List
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-bullets[bespoke-bullets]
+[.build.build-items]
```

Comment:
Actually, this was correct before. We should use `%build`, which is an alias (and best practice) for `.build.build-items`. You can see how it works here: https://github.com/asciidoctor/asciidoctor-bespoke/blob/master/templates/slim/helpers.rb#L147-L170